src/UserSpaceInstrumentation/CMakeLists.txt

```diff
@@ -33,6 +33,22 @@ target_link_libraries(UserSpaceInstrumentation PUBLIC
         OrbitBase
         CONAN_PKG::abseil)
 
+# Test Lib is merely used in UserSpaceInstrumentationTests below. A binary
+# libTestLib.so created from this target is checked in to testdata folder and
+# that binary is used to test the injection mechanism.
+add_library(TestLib SHARED)
```

Comment:
The name of the library should be expanded, for example to `UserSpaceInstrumentationTestLib`. If I'm not mistaken this still ends up in the `lib` folder and `libTestLib.so` is too generic.

Reply:
Yep, that's true. UserSpaceInstrumentationTestLib it is.